modules/tinydnsbackend/cdb.cc

```diff
@@ -22,33 +22,30 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
-#include "cdb.hh"
-#include <cdb.h>
-#include "pdns/misc.hh"
-#include "pdns/iputils.hh"
-#include <utility>
 
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "cdb.hh"
 
 CDB::CDB(const string &cdbfile)
 {
   d_fd = open(cdbfile.c_str(), O_RDONLY);
   if (d_fd < 0)
   {
-    g_log<<Logger::Error<<"Failed to open cdb database file '"<<cdbfile<<"'. Error: "<<stringerror()<<endl;
-    throw new PDNSException("Failed to open cdb database file '"+cdbfile+"'. Error: " + stringerror());
+    throw std::runtime_error("Failed to open cdb database file '"+cdbfile+"'. Error: " + stringerror());
   }
 
   memset(&d_cdbf,0,sizeof(struct cdb_find));
   int cdbinit = cdb_init(&d_cdb, d_fd);
   if (cdbinit < 0)
   {
-    g_log<<Logger::Error<<"Failed to initialize cdb structure. ErrorNr: '"<<cdbinit<<endl;
-    throw new PDNSException("Failed to initialize cdb structure.");
+    close(d_fd);
+    d_fd = -1;
+    throw std::runtime_error("Failed to initialize cdb structure. ErrorNt: '" + std::to_string(cdbinit) + "'");
```

Comment:
ErrorN_t_?

Reply:
This was the text in the existing code, yes, but unless I'm mistaken I replaced it in a later commit?